### DISCUSSION TITLE
Problem with free trial

### DISCUSSION THREAD

**Discussion by IzetDelibasic** [community/community#118562] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I activated the free trial of Copilot, but money was deducted from my bank card. How can I get my money back? I used it for three days and then deactivated it. What's the problem?

**Comment by 0xundef** [comment]
same issue. maybe there is no free 30 day trial

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
